build/pipelines/templates/build-console-audit-job.yml

```diff
@@ -8,7 +8,9 @@ jobs:
   variables:
     BuildConfiguration: AuditMode
     BuildPlatform: ${{ parameters.platform }}
-  pool: { vmImage: windows-2019 }
+  pool: "windevbuildagents"
+  # The public pool is also an option!
+  # pool: { vmImage: windows-2019 }

```

Comment:
Should you not mention that the `OpenConsoleCleanPch` flag should be on too if you go to the public pool?